pcdsdevices/mrco_motion.py

```diff
@@ -16,7 +16,8 @@ class MRCO(BaseInterface, Device):
     MRCO Motion Class
 
     This class controls motors fixed to the MRCO Motion system for the IP1
-    endstation in TMO.
+    endstation in TMO. Offset/Dir changes on Python/UI level have been 
```

Comment:
https://travis-ci.com/github/pcdshub/pcdsdevices/jobs/511197151#L205
```suggestion
    endstation in TMO. Offset/Dir changes on Python/UI level have been
```

Reply:
ok should be fixed now.